relayer/relays/execution/main.go

```diff
@@ -412,6 +416,26 @@ func (r *Relay) doSubmit(ctx context.Context, ev *contracts.GatewayOutboundMessa
 		"channelID":   types.H256(ev.ChannelID).Hex(),
 	})
 
+	destination, err := parachain.GetDestination(ev.Payload)
+	if err != nil {
+		return err
+	}
+
+	log.WithFields(log.Fields{
+		"destination": destination,
+		"payload":     common.Bytes2Hex(ev.Payload),
+	}).Info("extracted destination from message")
+	banned, err := r.ofac.IsBanned(ev.Raw.Address.Hex(), destination)
```

Comment:
Is `ev.Raw.Address.Hex()` equivalent to `msg.sender` or `tx.origin`. Because I think we need the latter. The original signer is OFAC banned.

Reply:
Addressed in https://github.com/Snowfork/snowbridge/pull/1313/commits/1e9558eccbc14a06e658fb86ac3c0733c6a79f61.